### ISSUE TITLE
[feature request] log x-forwarded-for header

### ISSUE THREAD

**Issue by helmut72** [alash3al/scraply#6]
I'm using a reverse proxy in front of scraply to use scraply (and all other webapps) with SSL. Please log x-forwarded-for header instead of the requesting ip if the header exists or add an option to enable/disable it. Thank you.

**Comment by alash3al** [comment]
for logging, do you mean the `scraply serve` stdout logging?
if yes, ok will check it and plan for it.

**Comment by helmut72** [comment]
Yes, for logging when `scraply serve` is running and log to stdout. Now it shows me the IP of my reverse proxy:
```
21:31:44 | 200 |   684ms |      172.18.0.1 | POST    | /extract 
```
